packages/vue/package.json

```diff
@@ -41,10 +41,11 @@
     "dist"
   ],
   "scripts": {
-    "build": "tsup",
+    "build": "tsup --onSuccess \"pnpm build:dts\"",
+    "build:dts": "vue-tsc --declaration --emitDeclarationOnly && rm -rf dist/components/__tests__",
```

Comment:
Can you tell vue-tsc to ignore the test files so that we don't have to run `rm -rf`?

Reply:
Updated! Got the idea [here](https://github.com/vuejs/language-tools/issues/3144#issuecomment-1986271761)